Assets/BossRoom/Scripts/Shared/Game/UI/Editor/NetworkLatencyWarning.cs

```diff
@@ -32,10 +32,10 @@ void Update()
                     case UnityTransport unityTransport:
                         // adding this preprocessor directive check since UnityTransport's simulator tools only inject latency in #UNITY_EDITOR or in #DEVELOPMENT_BUILD
 #if UNITY_EDITOR || DEVELOPMENT_BUILD
-                        SimulatorUtility.Parameters simulatorParameters = unityTransport.ClientSimulatorParameters;
-                        m_ArtificialLatencyEnabled = simulatorParameters.PacketDelayMs > 0 ||
-                            simulatorParameters.PacketJitterMs > 0 ||
-                            simulatorParameters.PacketDropPercentage > 0;
+                        var simulatorParameters = unityTransport.DebugSimulator;
```

Comment:
already bumped the transport channel on breaking changes here.

Reply:
Not a fan of this being modified on the prefab, but hey, this works for now.

Reply:
What do you mean prefab? This is just an API adjustement of something already there.

Reply:
So before, you'd configure latency through the "Netcode/Simulator Tools" window. Now, that comes through the "Debug Simulator" dropdown on the `Unity Transport` component on the NetworkManager prefab. So it's just prone to prefab commits if we're not careful.